lib/ace/edit_session.js

```diff
@@ -679,10 +679,10 @@ var EditSession = function(text, mode) {
     };
 
     /**
-    * Returns an array containing the IDs of all the markers, either front or back.
+    * Returns an object containing the all of the markers, either front or back.
```

Comment:
Squashing would be cleaner i guess, but i usually do not ask people to do that since it requires quite some effort for fairly small benefit.